### PR TITLE
OSDOCS-12362: 4.14.39 z-stream RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14

Issue:
[OSDOCS-12362](https://issues.redhat.com/browse/OSDOCS-12362)

Link to docs preview:
https://83959--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-39_release-notes

QE review:
N/A

Additional information:
Errata URLs will return 404 until go-live (10/23)

### CODE REVIEW

**Comment by obrown1205** [comment]
CC @wgabor0427 :)

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3384,6 +3384,54 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* The Insights Operator (IO) now collects data from the `haproxy_exporter_server_threshold` metric of Prometheus. (link:https://issues.redhat.com/browse/OCPBUGS-41918[*OCPBUGS-41918*])
+
+[id="ocp-4-14-39-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if a connection on port 9637 for Windows nodes was refused, the Kubelet Service Monitor threw a `target down` alert because CRI-O does not run on Windows nodes. With this release, Windows nodes are excluded from the Kubelet Service Monitor. (link:https://issues.redhat.com/browse/OCPBUGS-42603[*OCPBUGS-42603*])
+
+* Previously, when the Node Tuning Operator (NTO) was configured using PerformanceProfiles, it created an `ocp-tuned-one-shot systemd` service which ran prior to kubelet and blocked NTO execution. This prevented Podman from fetching the image. With this release, support for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env` is available. This allows Podman to pull NTO images into environments that need to use the http(s) proxy for out-of-cluster connections. (https://issues.redhat.com/browse/OCPBUGS-42567[*OCPBUGS-42567*])
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:3420]:
```suggestion
* Previously, when the Node Tuning Operator (NTO) was configured using the `PerformanceProfiles` specification, it created an `ocp-tuned-one-shot systemd` service which ran prior to kubelet and blocked NTO execution. This prevented Podman from fetching the image. With this release, support for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env` is available. This allows Podman to pull NTO images into environments that need to use the http(s) proxy for out-of-cluster connections. (https://issues.redhat.com/browse/OCPBUGS-42567[*OCPBUGS-42567*])
```

Comment (skrthomas) [reply release_notes/ocp-4-14-release-notes.adoc:3420]: Wondering if this could be more specific?

**Review by skrthomas — COMMENTED** [review]
Nice work. I only had one lil nit :)

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-14-release-notes.adoc:3420]: I like it :)

**Pull request merged**